.github/workflows/puppetize.yml

```diff
@@ -87,13 +87,15 @@ jobs:
         env:
           FORGE_TOKEN: ${{ secrets.FORGE_API_TOKEN }}
         run: |
+          Import-Module $env:ChocolateyInstall\helpers\chocolateyProfile.psm1
+          refreshenv
           Import-Module -Name PSDesiredStateConfiguration -Force
-          Import-Module -Name Puppet.Dsc -Force
+          Import-Module -Name ./src/Puppet.Dsc -Force
```

Comment:
one thing... should we revert this back to not use the local version of the module?

Reply:
going to wait until its released to change it back incase we need to release any dsc modules